paddle/fluid/operators/math/softmax.h

```diff
@@ -19,7 +19,8 @@ namespace paddle {
 namespace operators {
 namespace math {
 
-template <typename DeviceContext, typename T, bool is_test>
+template <typename DeviceContext, typename T, bool is_test,
```

Comment:
Suggestion just for the future reference. You can safely ignore it. 

Using the boolean `is_test` flag makes code a bit difficult to follow due to constant values `true` or `false` in the specializations of the class.

Maybe in future you could try to experiment with property type for inference defined along these lines:
``` struct is_test {};``` and use it whenever type specific for inference is used.

Reply:
Interesting suggesstion will look to it closer on my next PR related to softmax for inference